python/src/connection.rs

```diff
@@ -41,11 +41,20 @@ impl Connection {
 
 #[pymethods]
 impl Connection {
-    pub fn table_names(self_: PyRef<'_, Self>) -> PyResult<&PyAny> {
+    pub fn table_names(
+        self_: PyRef<'_, Self>,
+        page_token: Option<String>,
+        limit: Option<u32>,
+    ) -> PyResult<&PyAny> {
```

Comment:
Hmmm. I wonder if we would consider changing the opaque `page_token` parameter to `startOn` or `startAfter`. Since we are guaranteeing an alphabetical list of strings, it doesn't seem like we need the opaqueness, and having a more concrete parameter would be more useful.

For example, it makes it easier to get tables with a particular prefix, or to check for existence of a single table.

Reply:
I've renamed `page_token` to `start_after`/`startAfter` (I also added a lint rule for naming conventions since `page_token` in a typescript file was definitely wrong).

At the risk of bike shedding, I don't this really solves all problems.

 * `start_on` doesn't work well for paging because it forces you to grab the previous result
 * `start_after` doesn't work well for single table since it will skip the given item (it can work "ok" for prefix but its still a bit weird)

I've stuck with `start_after` because paging was the original intention.  Maybe we want to add a `filter` option in the future?

Reply:
`startAfter` sounds good to me. Yeah `filter` might be a better solution.